src/main/java/com/pikachu/purple/application/perfume/service/application/PerfumeDetailGetByPerfumeIdApplicationService.java

```diff
@@ -0,0 +1,37 @@
+package com.pikachu.purple.application.perfume.service.application;
+
+import com.pikachu.purple.application.mainAccord.service.domain.MainAccordDomainService;
+import com.pikachu.purple.application.perfume.port.in.PerfumeDetailGetByPerfumeIdUseCase;
+import com.pikachu.purple.application.perfume.service.domain.PerfumeDomainService;
+import com.pikachu.purple.application.perfume.service.domain.PerfumeNoteDomainService;
+import com.pikachu.purple.domain.mainAccord.MainAccord;
+import com.pikachu.purple.domain.perfume.Perfume;
+import com.pikachu.purple.domain.perfume.PerfumeDetail;
+import com.pikachu.purple.domain.perfume.PerfumeNote;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class PerfumeDetailGetByPerfumeIdApplicationService implements PerfumeDetailGetByPerfumeIdUseCase {
+
+    private final PerfumeDomainService perfumeDomainService;
+    private final PerfumeNoteDomainService perfumeNoteDomainService;
+    private final MainAccordDomainService mainAccordDomainService;
+
+    private final int MAX_SIZE = 5;
```

Comment:
p3) 변하지 않는 값이라면 매직 넘버로 표현하는건 어떨까요?
```suggestion
    private static final int MAX_SIZE = 5;
```